interactive_engine/executor/ir/graph_proxy/src/adapters/gs_store/details.rs

```diff
@@ -0,0 +1,570 @@
+//
+//! Copyright 2021 Alibaba Group Holding Limited.
+//!
+//! Licensed under the Apache License, Version 2.0 (the "License");
+//! you may not use this file except in compliance with the License.
+//! You may obtain a copy of the License at
+//!
+//! http://www.apache.org/licenses/LICENSE-2.0
+//!
+//! Unless required by applicable law or agreed to in writing, software
+//! distributed under the License is distributed on an "AS IS" BASIS,
+//! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//! See the License for the specific language governing permissions and
+//! limitations under the License.
+
+use std::fmt;
+use std::sync::atomic::{AtomicPtr, Ordering};
+
+use ahash::{HashMap, HashMapExt};
+use dyn_type::Object;
+use global_query::store_api::PropId;
+use global_query::store_api::{Edge as StoreEdge, Vertex as StoreVertex};
+use ir_common::{KeyId, NameOrId};
+use pegasus_common::downcast::*;
+
+use super::read_graph::{encode_runtime_prop_val, encode_runtime_property};
+use crate::apis::Details;
+use crate::apis::PropertyValue;
+
+/// LazyVertexDetails is used for local property fetching optimization.
+/// That is, the required properties will not be materialized until LazyVertexDetails need to be shuffled.
+#[allow(dead_code)]
+pub struct LazyVertexDetails<V>
+where
+    V: StoreVertex + 'static,
+{
+    // prop_keys specify the properties we would save for later queries after shuffle,
+    // excluding the ones used only when local property fetching.
+    // Specifically, in graphscope store, None means we need all properties,
```

Comment:
Have we fixed this? I think currently, in GraphScope Store, None means we do not need any property, and Some(vec![]) means we do not need any property. If so, please modify this comment (as well as for `LazyEdgeDetails ` )